data/load-seed-data.js

```diff
@@ -10,23 +10,23 @@ async function run() {
   try {
 
     await Promise.all(
-      cats.map(cat => {
+      items.map(item => {
         return client.query(`
-          INSERT INTO cats (name, type, url, year, lives, is_sidekick)
-          VALUES ($1, $2, $3, $4, $5, $6);
-        `,
-        [cat.name, cat.type, cat.url, cat.year, cat.lives, cat.isSidekick]);
+          INSERT INTO items (name, description, type, rarity, requires-attunement)
```

Comment:
i think you'll want `description` to be `desc`

Reply:
Thanks for all your help today!